### PR TITLE
Improves admin outfits, fixes magboots, adds special antags

### PR DESCRIPTION
**Pull request by Kyep**
Fixes:
- Admin Outfits that should have magboots (syndi op / freedomop / officer, NT VIP, solgov marine/LT, chrono legionaire, various hardsuit outfits) now get them. 
- Dressing certain mobs with no mind no longer generates a runtime in datacore.dm due no H.mind.
- It is no longer possible for NT Specops officers to spawn with no mindshield.

Tweaks:
- Syndi officers now get flashlights.
- Mime assassins no longer get a pistol/suppressor/pen. Instead, they get a radio uplink. They still get the cane gun/ammo.
- NT Navy Officer and NT Special Ops now show up exclusively under the 'as job...' section of the menu. Additionally, their mindshield and dust implants come pre-installed. They also get a special version of the survival box, that contains: a pre-filled double emergency air tank, seclite, combat knife, centcom bounced radio, door remote for all doors on the Cyberiad (but not centcom), death alarm implanter (manual use), combat nanites for healing, and an advanced pinpointer.
- NT Special Ops now get a full set of cybernetic implants (xray, anti-drop, anti-stun, nutri-plus). Yes, this means they will now be able to see people in the ERT starting area through the shutters.
- Dressing someone as an admin outfit will now assign their H.mind.assigned_role and H.job to the name of that outfit. This should make them show up correctly in things like Player Panel New.

New admin outfits:
- "NT Undercover Operative": undercover special ops officer, intended for use by NT assassinating mutinous Captains and other wet-work for which deathsquad armor is too obvious. Starts with specops-level gear, including a NT Saber SMG with silencer, all-door remote, etc.
- "Arch Wizard": a wizard with armor and a 50-use spellbook (effectively, a wizard with 5x normal spell buying power)

Deletions:
- Removed the syndicate bomber outfit. I added this myself, ages ago, and I have never seen anyone use it.

Refactors:
- Refactored the way implants work for outfits. Now both normal job outfits and admin outfits have the 'implants' and 'cybernetic_implants' lists (can be empty).

No changelog as this is entirely invisible to normal players.

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
sending a gimmick team does not have the option to equip people "as job", if you want to do that too

**Comment by Kyep** [comment]
That's really a separate PR, but yeah, I might do that, in a separate PR.

**Comment by DarkPyrolord** [comment]
👍

**Comment by Kyep** [comment]
Removed the ancient vampire, since I realized there was a better way of doing the same thing. Might re-add in a later PR, but it will be staying out of this one.

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -806,6 +806,29 @@
...
 
+
+/obj/item/weapon/storage/box/centcomofficer/
+	name = "officer kit"
+
+/obj/item/weapon/storage/box/centcomofficer/New()
+	..()
+	contents = list()
+	sleep(1)
+	new /obj/item/clothing/mask/breath( src )
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/storage/boxes.dm]: No spaces in these

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -806,6 +806,29 @@
 				return
 	return ..()
 
+
+/obj/item/weapon/storage/box/centcomofficer/
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/storage/boxes.dm]: No / after this

File: code/datums/outfits/outfit_admin.dm [hunk 3]
```diff
@@ -1130,6 +1087,38 @@
...
+		/obj/item/weapon/kitchen/knife/ritual  = 1,
+		/obj/item/clothing/suit/wizrobe/red = 1,
+		/obj/item/clothing/head/wizard/red = 1
+	)
+
+/datum/outfit/admin/wizard/arch/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	if(visualsOnly)
+		return
+	//if(H.mind)
```
Comment (tigercat2000) [inline code/datums/outfits/outfit_admin.dm]: No commented out code please

Comment (Kyep) [reply code/datums/outfits/outfit_admin.dm]: Deleted

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 2 above)
Comment (Kyep) [reply code/game/objects/items/weapons/storage/boxes.dm]: Deleted

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (Kyep) [reply code/game/objects/items/weapons/storage/boxes.dm]: Changed " src " to "src".

**Comment by Kyep** [comment]
@tigercat2000 All your requests are done.

File: code/datums/outfits/outfit.dm [hunk 4]
```diff
@@ -22,6 +22,8 @@
 	var/pda = null
 	var/internals_slot = null //ID of slot containing a gas tank
 	var/list/backpack_contents = list() // In the list(path=count,otherpath=count) format
+	var/list/implants = list()
```
Comment (Fox-McCloud) [inline code/datums/outfits/outfit.dm]: var/list/implants = null

File: code/datums/outfits/outfit.dm [hunk 5]
```diff
@@ -22,6 +22,8 @@
...
+	var/list/cybernetic_implants = list()
```
Comment (Fox-McCloud) [inline code/datums/outfits/outfit.dm]: var/list/cybernetic_implants = null

File: code/datums/outfits/outfit.dm [hunk 6]
```diff
@@ -89,6 +91,12 @@
 			var/number = backpack_contents[path]
 			for(var/i = 0, i < number, i++)
 				equip_item(H, path, slot_in_backpack)
+		for(var/path in implants)
+			var/obj/item/weapon/implant/I = new path(H)
+			I.implant(H)
+		for(var/path in cybernetic_implants)
```
Comment (Fox-McCloud) [inline code/datums/outfits/outfit.dm]:
```
	if(cybernetic_implants)
		for(var/implant_type in cybernetic_implants)
			var/obj/item/organ/internal/O = newimplant_type(H)
			O.insert(H)
```

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 7]
```diff
@@ -806,6 +806,29 @@
...
 	return ..()
 
+
+/obj/item/weapon/storage/box/centcomofficer
+	name = "officer kit"
+
+/obj/item/weapon/storage/box/centcomofficer/New()
+	..()
+	contents = list()
+	sleep(1)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/boxes.dm]: What. Why?

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 8]
```diff
@@ -806,6 +806,29 @@
...
+	new /obj/item/weapon/kitchen/knife/combat(src)
+
+	new /obj/item/device/radio/centcom(src)
+	new /obj/item/weapon/door_remote/omni(src)
+	new /obj/item/weapon/implanter/death_alarm(src)
+
+	new /obj/item/weapon/reagent_containers/hypospray/combat/nanites(src)
+	new /obj/item/weapon/pinpointer/advpinpointer(src)
+
+	return
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/boxes.dm]: No trailing returns.

File: code/datums/outfits/outfit_admin.dm [hunk 9]
```diff
@@ -1,6 +1,13 @@
 // Used for 'select equipment'
 // code/modules/admin/verbs/debug.dm 566
 
+/datum/outfit/admin/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```
Comment (Fox-McCloud) [inline code/datums/outfits/outfit_admin.dm]: I believe this is ok. @tigercat2000 confirm?

**Comment by Kyep** [comment]
@Fox-McCloud All your issues should be addressed.

**Pull request merged**